.travis.yml

```diff
@@ -43,13 +38,17 @@ install:
   - if [[ "$DB" = "postgresql" ]]; then pip install psycopg2; fi
 
   # Setup application
-  - if [[ "$DB" = "sqlite" ]]; then invoke create_settings; fi
-  - if [[ "$DB" = "postgresql" ]]; then invoke create_settings --database-type postgresql; fi
+  - pip install wger
+  - if [[ "$DB" = "postgresql" ]]; then wger create_settings --database-type postgresql; fi
+
 
 # Create test databases
 before_script:
   - if [[ "$DB" = "postgresq" ]]; then psql -c 'DROP DATABASE IF EXISTS test_wger;' -U postgres; fi
   - if [[ "$DB" = "postgresql" ]]; then psql -c 'CREATE DATABASE test_wger;' -U postgres; fi
+  - npm install -g gulp
+  - npm install --global gulp-cli
```

Comment:
Is there a reason why this was added?

Reply:
So that gulp lint can run. It can't run when installed from packages json.